doc/use_cli_effectively.md

```diff
@@ -94,44 +95,132 @@ Most update commands in the CLI feature the three generic arguments: `--add`, `-
     az vm update -g my_rg -n my_vm --add storageProfile.dataDisks @~/my_disk.json
     ```
 
-## Quoting Issues ##
+## Generic resource commands - `az resource`
+
+There may be cases where a service you are interested in does not have CLI command coverage. You can use the `az resource create/show/list/delete/update/invoke-action` commands to work with these resources. A few suggestions here:
```

Comment:
change "a few suggestions here" to "Here are a few suggestions:"

Reply:
```suggestion
There may be cases where a service you are interested in does not have CLI command coverage. You can use the `az resource create/show/list/delete/update/invoke-action` commands to work with these resources. Here are a few suggestions:
```